plugins/org.locationtech.udig.catalog.oracle/src/org/locationtech/udig/catalog/internal/oracle/OracleGeoResource.java

```diff
@@ -161,63 +170,34 @@ protected OracleResourceInfo createInfo( IProgressMonitor monitor ) throws IOExc
                 FeatureSource<SimpleFeatureType, SimpleFeature> source = dataStore
                         .getFeatureSource(typename);
                 ft = source.getSchema();
-                bounds = (ReferencedEnvelope) source.getBounds();
+                bounds = source.getBounds();
                 if (bounds == null) {
-                    CoordinateReferenceSystem crs = null;
-                    // GeometryDescriptor defaultGeometry =
-                    // source.getSchema().getGeometryDescriptor();
-                    crs = source.getSchema().getCoordinateReferenceSystem();
                     this.bounds = ReferencedEnvelopeCache.getReferencedEnvelope(info.getCRS());
-                    
-                    /*
-                    // no full table scan for you!
-                    bounds = new ReferencedEnvelope(new Envelope(), crs);
-                    FeatureIterator<SimpleFeature> iter = source.getFeatures().features();
-                    try {
-                        while( iter.hasNext() ) {
-                            SimpleFeature element = iter.next();
-                            if (bounds.isNull())
-                                bounds.init(element.getBounds());
-                            else
-                                bounds.include(element.getBounds());
-                        }
-                    } finally {
-                        iter.close();
-                    }
-                    */
                 }
-                // CoordinateReferenceSystem geomcrs = source.getSchema().getCRS();
-
-                // if(geomcrs!=null && !geomcrs.equals(CRS.decode("EPSG:4269"))){
-                // bounds = JTS.transform(bounds,CRS.transform(geomcrs,CRS.decode("EPSG:4269")));
-                // }else{
-                // if(geomcrs == null)
-                // System.err.println("CRS unknown for Shp");
-                // }
             } catch (Exception e) {
-                CatalogPlugin
-                        .getDefault()
-                        .getLog()
-                        .log(
-                                new org.eclipse.core.runtime.Status(
-                                        IStatus.WARNING,
-                                        "org.locationtech.udig.catalog", 0, Messages.OracleGeoResource_error_layerBounds, e)); //$NON-NLS-1$
+                CatalogPlugin.getDefault().getLog()
+                        .log(new org.eclipse.core.runtime.Status(IStatus.WARNING,
+                                "org.locationtech.udig.catalog", 0, //$NON-NLS-1$
+                                Messages.OracleGeoResource_error_layerBounds, e));
                 bounds = new ReferencedEnvelope(new Envelope(), null);
             }
 
             icon = Glyph.icon(ft);
-            keywords = new String[]{"postgis", //$NON-NLS-1$
-                    ft.getName().getLocalPart(), ft.getName().getNamespaceURI()};
+            keywords = new String[] { "postgis", //$NON-NLS-1$
```

Comment:
Hui, whats this? the class is within catalog.oracle module and keyword is postgis? I guess this is not valid .. and I'll create an issue for this